signbank/static/js/gloss_edit.js

```diff
@@ -220,7 +223,7 @@ function configure_edit() {
         var choices = choice_lists[$(this).attr('id')];
         var selected = [];
         for (var key in choices ) {
-            this.textContent.split(/,\s*/).indexOf(choices[key]) >= 0 && (selected.push(key));
+	     this.textContent.split(/\n\s*/).indexOf(choices[key]) >= 0 && (selected.push(key));
```

Comment:
I think this will break other fields that use this field type. We may need to accept it as an option instead,

Reply:
Could you elaborate please on what you mean by accept it as an option?
The only two fields affected at present are Usage and Semantic Fields, however, yes future devels would have to be made aware of how to present the field in the template. It is the template that makes the decision to split the field using commas. We could perhaps use semi-colons or something? Or HTML nbsp? - I think the JS can distinguish escape-codes.

Reply:
Agreed this is not necessary since it's fixed for usage as well

Reply:
After Google Meets conversation, we have decided that we will use the \n split and I will add a comment clarifying its use in the javascript for future developers.